src/md/src/DevGuides/retries.md

```diff
@@ -0,0 +1,67 @@
+# Retries reporting
+
+Some frameworks allow to retry `Test` execution according to provided conditions (TestNG) or just retry it predefined number of times.
+First execution is a simple `TestItem` and next ones are called `Retries`.
+In Report Portal these items are grouped and displayed as a sequence of executions for the same `Test`:
+
+![](/src/Images/devguide/retry/retry.png)
+
+When you click on the `retries` link you can switch between `Retries` and see their logs:
+
+![](/src/Images/devguide/retry/retry_switch.png)
+
+On the `Log view` you can also switch between `Retries`:
+
+![](/src/Images/devguide/retry/retry_view.png)
+
+## Retries reporting
+
+Common child `Test item` request ([check reporting dev guide](https://github.com/reportportal/documentation/blob/master/src/md/src/DevGuides/reporting.md)) looks like: 
+
+```shell
+curl --header "Content-Type: application/json" \
+     --header "Authorization: Bearer 039eda00-b397-4a6b-bab1-b1a9a90376d1" \
+     --request POST \
+     --data '{"name":"example step","startTime":"1574423237000","type":"step","launchUuid":"<launch uuid>","description":"Item that should be retried"}' \
+     http://rp.com/api/v1/rp_project/item/<parent uuid>
+```
+
+With body:
+
+```json
+{
+  "name": "example step",
+  "startTime": "1574423237000",
+  "type": "step",
+  "launchUuid": "<launch uuid>",
+  "description": "Item that should be retried"
+}
+```
+
+And response:
+
+```json
+{
+  "id": "uuid-of-the-first-step"
+}
+```
+
+Request for a `retry` looks the same, but has field `retry=true`:
+
+```json
+{
+  "name": "example step",
+  "startTime": "1574423237100",
+  "type": "step",
+  "launchUuid": "<launch uuid>",
+  "description": "Item that should be retried",
+  "retry": true
+}
+```
+
+To be displayed as a `retry` reported `Test item` should have the same `name`, `parentUuid`, `launchUuid` and `uniqueId` (if you provided it explicitly).
```

Comment:
Need to extend with with couple examples:

- what if `name` will be different, but `uniqueID` the same and `retry=true`?
- what if `name` will be different, but `uniqueID`the same and no `retry` param?
- what if user will send same `name` and same `uniqueID`, but no `retry` flag - will it be converted to  retry?
- what if we use **re-run** functionality of launch (which is just reusage of launch ID), and test items will be received with same `name`, same explicitly added `uniqueID`?

Reply:
1) Case added to the doc
2) Added
3) Added
4) There is no special behaviour for this case, because if it's a re-run this items will be added to the existing launch just as they would be added for non-rerun launch.